components/asio/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
I think we cannot distribute the `asio` component under Apache License. The library's original license is BSL 1.0: https://github.com/chriskohlhoff/asio/blob/master/asio/LICENSE_1_0.txt

Reply:
You are right. I've changed the license. Thank you for the review!